### PR TITLE
Update documentation with how to build a provider

### PR DESCRIPTION
**Pull request by alinbalutoiu**
Add steps with how to build a Python Provider.

Closes issue #31 and #32

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -44,10 +44,54 @@ cd Build-omi-script-provider/omi/Unix
...
+> Build-omi-script-provider/scriptprovider/output/bin/omigen_py schema.mof XYZ_Frog
+Creating "schema.py"
+Creating "mi_main.py"
+```
+The omigen_py tool will generate the files ```schema.py``` and ```mi_main.py```.
+
+The file mi_main.py will be edited in order to implement the custom python provider.
+Please refer to [Script Provider Getting Started Guide](/doc/ScriptProviderGettingStarted.pdf) for more details.
+
+After all the files have been updated, we need to copy ```schema.py``` and ```mi_main.py``` in a folder with the name of the class inside the root library dir of the OMI. For the developer build, this will be ```./Build-omi-script-provider/omi/Unix/output/lib```. If the OMI was installed with a package, the root library location is ```/opt/omi/lib```. Replace the path with the one that applies for your environment, in our case will be the first one since we built it from source.
```
Comment (EMumau) [inline README.md]: The name of the directory does not need to match the class name.  Which would be really difficult for providers that include multiple classes.

**Review by EMumau — CHANGES_REQUESTED** [review]
You should read this document and revise this one:  https://github.com/Microsoft/omi-script-provider/blob/master/doc/ScriptProviderGettingStarted.pdf

**Comment by JumpingYang001** [comment]
@alinbalutoiu , don't forget to update https://github.com/Microsoft/omi-script-provider/blob/master/doc/ScriptProviderGettingStarted.pdf too if need, I will merge this PR first, thanks.

**Review by JumpingYang001 — APPROVED** [review]
LGTM

**Comment by JumpingYang001** [comment]
@alinbalutoiu , I can't merge from this page since Eric was requesting changes, may be you need to update https://github.com/Microsoft/omi-script-provider/blob/master/doc/ScriptProviderGettingStarted.pdf, then ask Eric help to signoff, thanks.

**Comment by JumpingYang001** [comment]
@alinbalutoiu , ScriptProviderGettingStarted.pdf has very detailed information, I think you might be better document the things there. :)

**Comment by alinbalutoiu** [comment]
@JumpingYang001 most of the info is from ScriptProviderGettingStarted.pdf and I put in the README a resume of that one. @EMumau can you please review this again? Thanks!

**Review by EMumau — APPROVED** [review]

**Pull request merged**